R/fullCELLULARMAGPIE.R

```diff
@@ -206,12 +206,13 @@ fullCELLULARMAGPIE <- function(rev = 0.1, dev = "",
              subtype = "/co2/Nreturn0p5", # nolint
              lsu_levels = c(seq(0, 2.2, 0.2), 2.5), past_mngmt = "mdef",
              file = paste0("f31_grassl_yld.mz"), years = magYears, aggregate = FALSE)
-  calcOutput("PastureSuit",  subtype = paste("ISIMIP3bv2", "MRI-ESM2-0", "1850_2100", sep = ":"),
+  calcOutput("PastureSuit", datasource = "ISIMIP3bv2", climatetype = "MRI-ESM2-0:ssp126",
```

Comment:
climatetype and lpjml argument has to be set to the flexible arguments as they change with the preprocessing settings.

See e.g.: https://github.com/pik-piam/mrmagpie/blob/5b6edcabdcc51e52a944586258d038abaea9a3f6/R/fullCELLULARMAGPIE.R#L322

My guess how the call should look like:
 calcOutput("PastureSuit", climatetype = climatetype, lpjml =  lpjml[["natveg"]],
             file = paste0("f31_pastr_suitability_", ctype, ".mz"), years =  lpjYears, aggregate = "cluster")
             
Check which years make most sense (magYears or lpjYears) and the lpjml version ('natveg' or 'grass').